### PR TITLE
Fix update logic for Dr. CI

### PR DESCRIPTION
**Pull request by DanilBaibak**
Make available Dr. CI update feature also for domains.

### CODE REVIEW

**Review by malfet — APPROVED** [review]
It's missing update to the query on the rockset, isn't it?

File: torchci/rockset/commons/__sql/recent_pr_workflows_query.sql [hunk 1]
```diff
@@ -39,7 +43,8 @@ SELECT
     w.html_url,
     recent_shas.number AS pr_number,
     w.head_sha,
-    null as failure_line
+    null as failure_line,
+    recent_shas.sha_time
```
Comment (malfet) [inline torchci/rockset/commons/__sql/recent_pr_workflows_query.sql:47]: Why this change is needed?

File: torchci/rockset/commons/__sql/recent_pr_workflows_query.sql [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+    recent_shas as (
+        SELECT
+            p.head.sha as sha,
+            p.number as number,
+            PARSE_TIMESTAMP_ISO8601(push.head_commit.timestamp) as sha_time,
+        FROM
+            workflow_job j
+            JOIN commons.pull_request p ON j.head_sha = p.head.sha
+            JOIN commons.push push on push.head_commit.id = j.head_sha
+        where
```
Comment (malfet) [inline torchci/rockset/commons/__sql/recent_pr_workflows_query.sql:11]: Why this is needed?

**Review by malfet — CHANGES_REQUESTED** [review]
This makes quote a lot of changes to recent_pr_workflows other than adding repo name as parameter. Why those changes are needed?

File: torchci/rockset/commons/__sql/recent_pr_workflows_query.sql (hunk 1 above)
Comment (DanilBaibak) [reply torchci/rockset/commons/__sql/recent_pr_workflows_query.sql:47]: These changes came from Rщckset when I ran the command: `yarn node scripts/downloadQueryLambda.mjs commons recent_pr_workflows_query`. [Here is the query](https://console.rockset.com/lambdas/details/commons.recent_pr_workflows_query?tab=sql ).

File: torchci/rockset/commons/__sql/recent_pr_workflows_query.sql (hunk 2 above)
Comment (DanilBaibak) [reply torchci/rockset/commons/__sql/recent_pr_workflows_query.sql:11]: Same thing - I just pulled the changes from Rockset and committed them.

**Review by malfet — APPROVED** [review]

File: torchci/rockset/commons/__sql/recent_pr_workflows_query.sql (hunk 2 above)
Comment (malfet) [reply torchci/rockset/commons/__sql/recent_pr_workflows_query.sql]: Figured it out offline: somebody made changes to the query, which were not deployed to production and as result got pulled into this PR. Updated fixed it.

**Review by malfet — APPROVED** [review]
Looks good to me
